tests/test_data_bleach.py

```diff
@@ -0,0 +1,70 @@
+import json
+import os
+import unittest
+import uuid
+from pathlib import Path
+
+from scripts.data_bleach import main, PUBLIC_FIELDS
+
+
+class TestDataBleach(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.test_path = Path(__file__).parents[0]
+        cls.test_file = cls.test_path / 'test_data/bleach_test_data.json'
+        cls.out_file = cls.test_path / f'{str(uuid.uuid4())}.tmp.json'
+
+    def setUp(self):
+        with self.assertRaises(FileNotFoundError):
```

Comment:
What do you think of having `setUp` remove the file (if it is there) as is done in `tearDown`?
That way if the file is there, it will be removed and the test can proceed rather than stopping.
Given the name of the `out_file` contains a unique UUID, I don't think there is concern with interfering with an existing test execution.

Reply:
I think that's a fine implementation, but the reason that I made it like this was so that if the file isn't getting deleted, then the tests will fail so that you know something is wrong. To me this is better than adding the redundancy of trying to delete the file in multiple places.

Either way it won't affect how things work all that much.

Reply:
Sounds good.